test/testcafe/spec/FailureRedirect_spec.js

```diff
@@ -25,7 +26,7 @@ test.requestHooks(userNotAssignedMock)('oauth: shows the error message', async t
     redirectUri: 'http://totally-fake'
   });
   await terminalPage.waitForErrorBox();
-  await t.expect(terminalPage.getErrorMessages().getTextContent()).eql('You are not allowed to access this app. To request access, contact an admin.');
+  await t.expect(terminalPage.getErrorBoxText()).eql('You are not allowed to access this app. To request access, contact an admin.');
```

Comment:
I updated this because the `getErrorMessages()` function used data-se instead of the role. They both point to the same alert box but the new function is cross compatible